ntpoffset.py

```diff
@@ -24,23 +24,29 @@ def warn(msg):
 class NtpOffsetConfigException(Exception):
     pass
 
-
 class NtpOffset(object):
     PLUGIN_NAME = 'ntpoffset'
 
     def __init__(self, config=None):
         self.pool = None
+        self.absolutes = False
         self.ntp_client = ntplib.NTPClient()
         if config:
             self.config(config)
 
     def config(self, conf):
-        entries = [node for node in conf.children if node.key == 'pool']
+        for node in conf.children:
+          if node.key == 'pool':
+            config_pool = node
+          elif node.key == 'absolutes':
+            self.absolutes = str(node.values[0]).lower() in ['true', 'yes', 't', '1', 'on']
+          else:
+            warn('unknown config key {key}'.format(key=node.key))
 
         try:
-            self.pool = entries[0].values[0]
+            self.pool = config_pool.values[0]
 
-        except IndexError:
+        except UnboundLocalError:
```

Comment:
This would be clearer now by testing `config_pool` first (it's not as Pythonic, but it's more readable, IMO), eg:
```python
if config_pool:
    self.pool = config_pool.values[0]
else:
    raise NtpOffsetConfigException('No pool specified')
```